app/containers/WriteReviewPage/index.js

```diff
@@ -14,6 +14,7 @@ import injectSaga from 'utils/injectSaga';
 
 import * as actions from './actions';
 import * as selectors from './selectors';
+import * as constants from './constants';
```

Comment:
This seems strange to me. I'm pretty positive there isn't an effect on performance by using `import *`, but it seems like overkill when only using one or two constants. More importantly, isn't it a violation of Redux to not be using a selector to get store data?

Reply:
It's not reaching into the store here, those constants are simply strings or whatever, unique keys to access things in the store

What i'd like to do instead here is have constants have like
````export default const constants = {
  ConstName1: "value 1",
  ....
```

Good call on affecting perf but I bet it's very minor -- kinda lazy about these imports but it's probably good to be disciplined about them